src/exchange_adapters/bitget.ts

```diff
@@ -0,0 +1,104 @@
+import { Exchange } from '../utils'
+import { BaseExchangeAdapter, ExchangeDataType, Ticker, Trade } from './base'
+
+export class BitgetAdapter extends BaseExchangeAdapter {
+  baseApiUrl = 'https://api.bitget.com/api'
+
+  readonly _exchangeName = Exchange.BITGET
+  // bitget.com cert
```

Comment:
can you pls make this consistent so use the second license

Reply:
done